internal/servers/worker/worker.go

```diff
@@ -265,11 +266,20 @@ func (w *Worker) getSessionTls(hello *tls.ClientHelloInfo) (*tls.Config, error)
 	ctx := w.baseContext
```

Comment:
might be nice to have a test for the unexported func, given how buried it is during runtime...